### ISSUE TITLE
Opens up multiple black windows and throws a snap error

### ISSUE THREAD

**Issue by mmphego** [johannesjo/super-productivity#239]
This started with the new update

On launch it opens up multiple blank windows and upon pomodoro start or task completion it throws out the error, See snapshot below

![Screenshot_2019-10-14_08-42-09](https://user-images.githubusercontent.com/7910856/66733120-c753b880-ee5e-11e9-9835-9658a7d05572.png)

Version: 2.13.0

snap --version
```
snap    2.42
snapd   2.42
series  16
ubuntu  18.04
kernel  5.0.0-31-generic
```

**Comment by johannesjo** [comment]
This should be fixed with 2.13.2 and should only occur in 2.13.0 if google drive sync is enabled but fails for some reason. Please let me know if this isn't the case for you.

**Comment by mmphego** [comment]
Resolved with the latest version, had to reinstall.

Thanks for the great app, it's making a difference.
